test/module.test.js

```diff
@@ -1,13 +1,18 @@
+jest.setTimeout(60000)
+
 const { Nuxt, Builder } = require('nuxt-edge')
 
+const config = require('./fixture/nuxt.config')
+config.dev = false
+
 describe('module', () => {
   let nuxt
 
   beforeAll(async () => {
-    nuxt = new Nuxt(require('./fixture/nuxt.config'))
+    nuxt = new Nuxt(config)
     await nuxt.ready()
     await new Builder(nuxt).build()
-  }, 60000)
+  })
```

Comment:
Timeout can be passed as second argument here

Reply:
This is being used `jest.setTimeout(60000)`

Reply:
Is it in latest template?

Reply:
(If so, we have to also update it. 60sec is too much and only required for build)

Reply:
Yes, it is. But in the template build and listen